util/install/ansible-bootstrap.sh

```diff
@@ -137,7 +137,7 @@ if [[ "true" == "${RUN_ANSIBLE}" ]]; then
     make requirements
 
     cd "${CONFIGURATION_DIR}"/playbooks/edx-east
-    "${PYTHON_BIN}"/ansible-playbook edx_ansible.yml -i '127.0.0.1,' -c local -e "configuration_version=${CONFIGURATION_VERSION}"
+    "${PYTHON_BIN}"/ansible-playbook edx_ansible.yml -i '127.0.0.1,' -c local -e "configuration_version=${CONFIGURATION_VERSION}" -e "edx_ansible_source_repo=${CONFIGURATION_REPO}" 
```

Comment:
This change was part of https://github.com/Microsoft/edx-configuration/pull/155 and shouldn't be included here.

Reply:
YES, I don't why this been coming.. Its not part of my commits that i have sent for review.